shinydashboard/lantern/functions/endpoints.R

```diff
@@ -153,9 +154,10 @@ get_fhir_resource_count <- function(fhir_resources_tbl){
     group_by(type, fhir_version) %>% count() %>% rename(Resource = type, Endpoints = n)
 }
 
-# @TODO Repeat above, first figure out what the fhir_resources_tbl is
 get_capstat_fields_count <- function(capstat_fields_tbl) {
-  # @TODO this is definitely not right
   res <- capstat_fields_tbl %>%
-    group_by(field, fhir_version) %>% count() %>% rename(Fields = field, Endpoints = n)
+    group_by(field, exist, fhir_version) %>%
+    count() %>%
+    filter(exist == "true") %>%
+    rename(Fields = field, Endpoints = n)
```

Comment:
I'd recommend removing the exist column from the result here by adding `ungroup() %>% select(-exist)` and then you don't need to worry about hiding a column that you aren't using anyway.